### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by cartige**

### CODE REVIEW

**Review by PriscilliaAmmeux — DISMISSED** [review]

File: frontend/package.json [hunk 1]
```diff
@@ -11,11 +11,15 @@
   },
   "dependencies": {
     "leaflet": "^1.9.2",
+    "localforage": "^1.10.0",
```
Comment (LoicBrassart) [inline frontend/package.json]: Wat?

File: frontend/package.json [hunk 2]
```diff
@@ -11,11 +11,15 @@
...
+    "match-sorter": "^6.3.1",
```
Comment (LoicBrassart) [inline frontend/package.json]: What's this one ?

File: frontend/package.json [hunk 3]
```diff
@@ -11,11 +11,15 @@
...
     "prop-types": "^15.8.1",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "react-icons": "^4.6.0",
     "react-leaflet": "^4.1.0",
+    "react-router-dom": "^6.4.3",
+    "sort-by": "^1.2.0",
```
Comment (LoicBrassart) [inline frontend/package.json]: And this ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Plz remove useles imports in package.json

**Review by PriscilliaAmmeux — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
Much better, no more Evils

**Pull request merged**